### ISSUE TITLE
System information not updated after FSD Jump, Scan values tab still show the previous system values

### ISSUE THREAD

**Issue by patmc6** [EDDiscovery/EDDiscovery#3492]
Hello

- launch EDD
- launch Elite Odyssey
- takeoff from Shinrarta Dezarta/Jameson
- FSD Jump to FIANNA
- full system scan, and long range scan each planet

-> EDD system info still shows Shinrarta Dezarta

EDD trace log : [Trace_2024-03-09-13-47-06.0.log](https://github.com/EDDiscovery/EDDiscovery/files/14546369/Trace_2024-03-09-13-47-06.0.log)

The planets names (ex: "Fianna 1") and local fleet carriers are visible in the game log file :

> { "timestamp":"2024-03-09T13:06:17Z", **"event":"FSSSignalDiscovered", "SystemAddress":7268292765097, "SignalName":"ILHA DOS PRAZERES H9V-G2B", "SignalType":"FleetCarrier"**, "IsStation":true }
> { **"timestamp":"2024-03-09T13:06:18Z", "event":"Scan", "ScanType":"Detailed", "BodyName":"Fianna 1"**, "BodyID":16, "Parents":[ {"Star":0} ], "StarSystem":"Fianna", "SystemAddress":7268292765097, "DistanceFromArrivalLS":951.149252, "TidalLock":false, "TerraformState":"", "PlanetClass":"Gas giant with ammonia based life", "Atmosphere":"", "AtmosphereComposition":[ { "Name":"Hydrogen", "Percent":74.219093 }, { "Name":"Helium", "Percent":25.734842 } ], "Volcanism":"", "MassEM":528.650024, "Radius":76137120.000000, "SurfaceGravity":36.348353, "SurfaceTemperature":115.815559, "SurfacePressure":0.000000, "Landable":false, "SemiMajorAxis":285134249925.613403, "Eccentricity":0.000121, "OrbitalInclination":1.800161, "Periapsis":12.421946, "OrbitalPeriod":160612946.748734, "AscendingNode":-52.267340, "MeanAnomaly":247.687647, "RotationPeriod":79039.099033, "AxialTilt":-0.485043, "Rings":[ { "Name":"Fianna 1 A Ring", "RingClass":"eRingClass_Icy", "MassMT":1.0499e+12, "InnerRad":1.2563e+08, "OuterRad":2.9636e+08 } ], "ReserveLevel":"PristineResources", "WasDiscovered":true, "WasMapped":true }

**Comment by Eahlstan** [comment]
Please screenshot your main history panel and then the event filter of that panel.

**Comment by patmc6** [comment]
- scan current system GD 319
- exit/relaunch EDD, while in game: it shows GD 319 correctly
- FSD jump to Lawd 52
- scan (honk)
- EDD still shows the previous system instead of Lawd 52
- screenshot
![image](https://github.com/EDDiscovery/EDDiscovery/assets/39314216/8701d3bf-4292-4e13-a04d-4015a3d3c05d)

**Comment by Eahlstan** [comment]
Hm. Sounds like a hanging issue of the file watcher. Some people had that in the past, never found out why.
Please hover your mouse over the double line at the top of EDD to unroll the menu you hid. You'll see a dropdown menu with your commanders to the left and two circling arrows next to it. Please click those arrows, this causes a refresh and fixed it in the very rare cases we had so far.

Did you already restart Windows? Did anything change on your system when this started to happen?

>> "All" is checked, but scan stays unchecked, even if I repeatedly click "All"

That's fine, "Scan" is a subset of events, if you'd click that "All" would be void and only events fitting that subtype would get selected.

**Comment by patmc6** [comment]
> Please click those arrows, this causes a refresh and fixed it in the very rare cases we had so far.
I see the refresh message in the bottom right log, but it did not fix it

> Did you already restart Windows? Did anything change on your system when this started to happen?
Rebooting does not fix it.
I have had the issue for a few weeks on a desktop windows 10 (currently on it now). I migrated the DB from my former windows 7 1 year ago (which did not have the issue); I possibly had the issue without noticing it.
I also have it on a windows 11 laptop, fresh install of EDD, I did not copy the DB or settings.

Maybe some windows setting is messing with the file watchers ? Both computers have OOSU 10, but the filters look fairly settings (I unset the 3 confidentiality settings I had, no change)

**Comment by patmc6** [comment]
> > Please click those arrows, this causes a refresh and fixed it in the very rare cases we had so far.
> 
> I see the refresh message in the bottom right log, but it did not fix it

Actually... When I click refresh, let it finish (checking the status bar messages), then use the filter menu to switch between "show the last 10 entries" or "last 20 entries", then the history display is updated (I keep going in and out of supercruise to generate events). If if only refresh, or if I only changes the filter, it does not work. Looks like 2 distinct problems (watcher vs display update).

**Comment by Eahlstan** [comment]
Please upload the latest trace log (like the one you provided in the first post).

**Comment by patmc6** [comment]
> Please upload the latest trace log (like the one you provided in the first post).

I have just added them to the previous comment.

I tried something else:
- start EDD
- tail -f the current EDD trace log (not the frontier journal)
- launch Elite, enter solo mode --> the trace is updated as soon as the I "get in the cockpit" after the loading screen
- open the station services menu, close it later --> Trace is immediately updated:
```
127: 2024-03-09T22:50:03.791Z: UI Event 09/03/2024 22:50:03 GUIFocus : StationServices
128: 2024-03-09T22:50:18.808Z: UI Event 09/03/2024 22:50:18 GUIFocus : NoFocus
```
- exit the game @22:56Z--> Trace is updated
- However EDD history tab still shows the event since the first step (shutdown game from my previous session 7 hours ago) 
![image](https://github.com/EDDiscovery/EDDiscovery/assets/39314216/796d25cb-afc2-4ea6-a2da-0ed4cb2135ba)
- exit EDD
[Trace_2024-03-09-23-33-48.0.log](https://github.com/EDDiscovery/EDDiscovery/files/14548056/Trace_2024-03-09-23-33-48.0.log)

So I suppose the file watchers on frontier journals are ok, the issue lies somewhere else

**Comment by Eahlstan** [comment]
Those are UI events taken from status.json, there's no sign that EDD reads any real event from the actual journal files.
That's the exact same strange behaviour seen before...

I doubt that it's because of that, but can you switch EDD back to english? Are you starting EDD with admin rights? Did you try temporary deactivating your security software? (Disclaimer: deactivating your security software is not a recommended thing to do.) Did you check if any of the files you copied from your old PC got locked by Windows? Right click -> properties -> see if there's a security warning saying "this file is from another PC" with an unchecked "allow" checkbox next to it. Did you only copy the user DB or the old journal files in SavedGames too?

I did read that you said it's the same on your Windows 11 with a completely fresh installation, but can you still do this on your Windows 10 machine?
- backup your user and system DB
- uninstall EDD
- delete EDDs folder in %localappdata%
- reinstall EDD and let it build a fresh user DB
- test if it works

**Comment by robbyxp1** [comment]
Without further information, this ticket is going nowhere. Timer is now ticking!

**Comment by patmc6** [comment]
I switched EDD to English. It's started without admin rights.
I have just tried to run it as admin, same behaviour (still showing the previous system name). Same thing if I temporarily disable windows defender anti-virus/threat protection

> Did you check if any of the files you copied from your old PC got locked by Windows? Right click -> properties -> see if there's a security warning saying "this file is from another PC" with an unchecked "allow" checkbox next to it.

I don't see any warning in the files properties.

> Did you only copy the user DB or the old journal files in SavedGames too?

Both, then I recursively changed the ownership (+ gave read&execute/modify/write permissions) of the whole profile to my windows 10 account  (otherwise the owner id of the copied files was still the one of the old windows 7)

>backup your user and system DB
uninstall EDD
delete EDDs folder in %localappdata%
reinstall EDD and let it build a fresh user DB
test if it works

It works ! The system name is updated immediately after the jump, the planet names are added when I scan them
I changed most of the settings to what they originally were (tabs, api keys, theme, etc...) still working.


Out of curiosity : if I switch back to the %localappdata%\EDDiscovery I backed up, EDD does not see jumps /FSS as before. But **if copy the new EDDUser.sqlite into the old %localappdata%\EDDiscovery** (keeping the same EDDSystem.sqlite) , it works (I tried removing the leftovers / overwriting the different files or directories before the user DB, no luck)

I'll try later to with the windows 11 PC

**Comment by Eahlstan** [comment]
Well obviously something went wrong when copying the old things. Whatever it is, it's a Windows (or one of the installed programs) issue/behaviour and there's nothing EDD can do about it.
Glad it's working now though.

**Comment by patmc6** [comment]
a few last details

**if anyone else stumbles on the same issue, rebuilding the user DB EDDUser.sqlite fixed it**

I booted windows 7 on the same PC, ran EDD (same version) + Elite on it --> EDD history is updated correctly.
I booted windows 10, copied EDDUser.sqlite from windows 7, run EDD + Elite --> EDD history does not update anymore.
I restored EDDUser.sqlite from yesterday --> all good again. So something does go wrong when importing the user DB from the former OS. Maybe add an advice in https://github.com/EDDiscovery/EDDiscovery/wiki/FAQ#moving-edd-to-a-new-computer to "backup/ rebuild your user DB when migrating to new OS or PC if something seems wrong " ?

More importantly, thank you for your help !
